src/util/mod_string.c

```diff
@@ -162,7 +162,8 @@ char *strtok_r(char * __restrict s1, const char * __restrict s2, char ** __restr
                 *p++ = 0;
             }
         } else {
-            p = s = NULL;
+            s = NULL;
+            p = NULL;
```

Comment:
I don't recall off-hand actually.  Both are equally valid and I might have just been in a mood at that point :package: